### ISSUE TITLE
[Bug]: Netdata packages installed by kickstart should have the same netdata.conf file

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#12404]
### Bug description

deb and rpm packages use custom [netdata.conf](https://github.com/netdata/netdata/blob/master/system/netdata.conf) file. The file is no longer needed after #12257 and #12256 issues have been fixed.

The goal is to have the same experience regardless of the installation method (native packages, static, build).

### Expected behavior

deb/rpm packages shouldn't use a custom configuration file. The config should be downloaded from Netdata (/netdata.conf) after installation (the same we do for static/build installs).

### Steps to reproduce

1. Install deb/rpm Netdata package.
2. Check `/etc/netdata/netdata.conf`

### Installation method

kickstart.sh

### System info

```shell
n/a
```

### Netdata build info

```shell
Version: netdata v1.33.1-164-g7750bb51c
```

**Comment by Ferroin** [comment]
I’m not 100% certain if this is still an issue at the moment or not.

**Comment by ilyam8** [comment]
@Ferroin native packages use this [netdata.conf](https://github.com/netdata/netdata/blob/master/system/netdata.conf). Downloading `netdata.conf` is not a part of native packages installation.

**Comment by Ferroin** [comment]
The only resolution for this that gives us 100% consistency across all install types and installation conditions is to just _not_ fetch the config on any install type, and expand the default file to better explain things (say, point users at the actual documentation instead of blindly assuming they can find it). Doing so will, of course, surprise some users _at first_, but over a few months that should die down, and doing so would both simplify the install process and make us far more consistent with how other software handles their config files.

If we choose to go the route of fetching the config even for package installs, that will still not make things consistent because:

- Installs where the agent is not started will not fetch the config, because they by definition cannot do so (this is already the case, and is not something we can fix without a massive redesign of how the agent configuration works).
- Installs that bypass the kickstart script and manually set up native packages will not fetch the config (because I am _not_ maintaining that code in three different places, especially when getting it wrong in two of the three places means that user configuration gets overwritten on updates).

And it would also have the issue of the package manager not removing the config file even in some cases when it does currently, because the PM will see it as having been modified by the user.
